syntax = "proto2";
package sweep_msgs.msgs;

message sweep
{
  required float time_msec       	= 1;
  required float min_distance      	= 2;
  required float max_distance  	 	= 3;
  required float current_distance     	= 4;
  required float degrees     	= 5;
}
